docs/eos-learn/blockchain-security.md

```diff
@@ -5,4 +5,10 @@ sidebar_label: Security
 ---
 ## Exploiting vulnerabilities in Blockchain
 
+The blockchain is a set of emerging technologies that are presented as alternatives to centralized systems. A notable example is that they seek to replace traditional banking systems through the use of cryptocurrencies without centralized control, as in the case of the well-known Bitcoin. They also have other important applications, such as in the traceability of assets, the control of corruption, and even democratic systems. Some of these capabilities are implemented through "smart contracts", basically software that runs on blockchains.
```

Comment:
Blockchain not "The blockchain" 

did we already correct this?

Reply:
I think we had, it may be a problem with my local branch

Reply:
seems like you have a merge conflict